lib/actions-oclif/login.ts

```diff
@@ -166,7 +171,7 @@ ${messages.reachingOut}`);
 		// Web
 		else if (loginOptions.web) {
 			const auth = await import('../auth');
-			await auth.login();
+			await auth.login({ port: loginOptions.port || 0 });
```

Comment:
Is this `|| 0` actually necessary? Also is it better as `?? 0` if you're specifically targetting nullish

Reply:
Removed -- it was not necessary. 👍